test/e2e_node/oomkiller_linux_test.go

```diff
@@ -59,7 +59,7 @@ var _ = SIGDescribe("OOMKiller for pod using more memory than node allocatable [
 	}
 
 	for _, testCase := range testCases {
-		runOomKillerTest(f, testCase, KubeReservedMemory)
+		runOomKillerTest(f, testCase, KubeReservedMemory, "UnlimitedSwap")
```

Comment:
One thing that is a bit odd about this. SwapBehavior is a KEP feature that is turned off. I think we still allow setting it. For this test, I think you are running these with swap turned off so this feature is ignored.

In the case of UnlimitedSwap, we are actually ignoring this field. Maybe we could ignore setting this field unless its LimitedSwap.

Reply:
@chansuke please update this also.